cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/DatabaseManagementTest.scala

```diff
@@ -65,6 +65,17 @@ SHOW DATABASE myDatabase
 
 List information about the database `myDatabase`.
 
+###assertion=show-one
+//
+
+SHOW DATABASES
```

Comment:
Where you added this we get the order:
```
SHOW DATABASES
SHOW DATABASE name
SHOW DATABASES YIELD ...
SHOW DEFAULT DATABASE
```
Not sure I like that, I'd say either move the new one last or second place (so either you list all different `SHOW` commands first and then the `YIELD`/`WHERE` or collect all `SHOW DATABASES` versions together before the other `SHOW` command versions)

Reply:
To follow the order used for `SHOW ROLES` it should be in second place:
```
SHOW DATABASES
SHOW DATABASES YIELD ...
SHOW DATABASE name
SHOW DEFAULT DATABASE
```

Reply:
Updated!